enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -180,6 +180,16 @@ This controller shapes the static pod manifest itself.  It has some unique featu
     It waits until it is able to know that it is in the member list before trying to start.
     When it does this, it is able to determine the member list to use to launch.
 
+### Replacing single lost master
```

Comment:
Alternative pitch

**How to recovery one etcd member report unhealthy in clusteroperator status**
The `Degraded` condition is true and indicating a single unhealthy member.  First determine which of these cases you're in.

1. the node in question is down and expected to come back up.  In this case, do nothing.
2. the node is up and generally healthy, but etcd is unhealthy for some reason.  In this case, go to the "forcing an existing member to rejoin"
3. the node is down and not expected to come back.  In this case, go to the "removing an etcd member that will never return" followed by "adding a new master node"

**forcing an existing member to rejoin**

1. `oc debug node/affectected-node`
2. `mv /etc/kubernetes/manifests/etcd-pod.yaml /backuplocation`
3. `rm -rf /var/whatever/memberdir/thingy`
4. `oc -n openshift-etcd rsh some-etcd-pod`
5. `etcdctl remove member the-bad-one`
6. `oc patch forceredpeloymment=reason-time`